### PR TITLE
Adds operating instructions

### PR DESCRIPTION
**Pull request by seemethere**
Signed-off-by: Eli Uriegas <eli.uriegas@docker.com>

### CODE REVIEW

File: deb/README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Building your own Docker deb package
+
+`.deb` packages can be built from this directory with the following syntax
+
+```shell
+make ENGINE_DIR=/path/to/engine CLI_DIR=/path/to/cli deb
```
Comment (thaJeztah) [inline deb/README.md]: I take it `engine` and `cli` are the source-repo's? 😇

File: deb/README.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+```shell
+make ENGINE_DIR=/path/to/engine CLI_DIR=/path/to/cli ubuntu
+```
+
+If you want to specify a specific distro version:
+
+```shell
+make ENGINE_DIR=/path/to/engine CLI_DIR=/path/to/cli ubuntu-xenial
+```
+
```
Comment (thaJeztah) [inline deb/README.md]: What could be added here is;

- an explanation where the artefacts end up
- perhaps a full example;

```bash
git clone -b <branch-foo> https://github.com/docker/docker-ce.git


make ENGINE_DIR=docker-ce/components/engine CLI_DIR=docker-ce/components/cli ubuntu
```

I'm not sure if we should show how to use github.com/moby/moby, but open to suggestions for that one 👍

File: deb/README.md (hunk 1 above)
Comment (seemethere) [inline deb/README.md]: Yup! I'll add a note about that

File: deb/README.md (hunk 2 above)
Comment (seemethere) [inline deb/README.md]: Sure, I'll make sure add a note about artifacts,  and a more full fledged example as well

**Pull request merged**